vllm/model_executor/layers/attention.py

```diff
@@ -253,8 +253,10 @@ def __init__(
         max_position: int = 8192,
         base: int = 10000,
         num_kv_heads: Optional[int] = None,
+        is_neox_style: bool = True,
```

Comment:
Nit: Do you think this will be more clear? 
```suggestion
        style: str = "neox",  # Options: ["neox", "gptj"]
```

Reply:
To my knowledge, there are only two types of RoPE in terms of how they rotate the query and key vectors (To my understanding, the rope scaling stuffs use the GPT-NeoX RoPE). I think we can change the interface after we find more RoPEs to support.